democracy/views/utils.py

```diff
@@ -199,3 +203,112 @@ def to_internal_value(self, data):
             else:
                 raise ValidationError(_('Image size should be smaller than {} bytes.'.format(settings.MAX_IMAGE_SIZE)))
         raise ValidationError(_('Invalid content. Expected "data:image"'))
+
+
+class TranslatableSerializer(serializers.Serializer):
+    """
+    A serializer for translated fields.
+
+    translated_fields must be declared in the Meta class.
+    By default, translation languages obtained from settings, but can be overriden
+    by defining translation_lang in the Meta class.
+    """
+
+    def __init__(self, *args, **kwargs):
+        assert hasattr(self.Meta, 'translated_fields'), 'Improperly configured: Need to specify translated_fields'
```

Comment:
`translated_fields` have the format `{field: {"en": "something", "fi": "..."}}`. If the field is not in the `translated_fields` (despite having a translation), the serialization would be `{field: "something"}`.

This might not very useful though. I can remove it if it is useless and makes things too complicated.